x-pack/legacy/plugins/siem/public/components/ml/score/translations.ts

```diff
@@ -35,6 +35,6 @@ export const NARROW_TO_THIS_DATE_RANGE = i18n.translate(
   }
 );
 
-export const TOP_ANOMALY_SUSPECT = i18n.translate('xpack.siem.ml.score.topAnomalySuspectTitle', {
-  defaultMessage: 'Top Anomaly Suspect',
+export const ANOMALOUS_ENTITY = i18n.translate('xpack.siem.ml.score.anomalousEntityTitle', {
+  defaultMessage: 'Anomalous Entity',
```

Comment:
I'm good with the change from `Anomaly Suspect` to `Anomalous Entity`, but isn't it technically still the `Top` entity? Should this be `Top Anomalous Entity`? The only reason I ask is because I don't want to give the impression that it's the _only_ anomaly.

Reply:
Good question @MichaelMarcialis I think the answer depends on the user's context, and whether and how that context is maintained in the current view.

**The argument against including "top":**
To get here, the user has already clicked on the _top_ anomaly score for that particular job, and now they're looking at the details about that _top-scoring anomaly_ for that job.  Once that context is set, there is only one anomalous entity, so no need to include "top."  In fact, including "top" might imply that there were multiple anomalous entities for this top scoring anomaly, might not be true.

In fact, that context is reinforced at the top of the detail display with "Max Anomaly Score"

![image](https://user-images.githubusercontent.com/22619280/61726056-fadee500-ad3e-11e9-8e8a-9af7ecffab5e.png)

BTW, the ML team does not take a stand on this labeling in their Anomalies table at the bottom of the Anomaly Explorer view, instead labeling the column "found for".
![image](https://user-images.githubusercontent.com/22619280/61725955-c3703880-ad3e-11e9-8ca6-a44034a62bba.png)

So I'd vote to leave it out.  Does this make sense?

Reply:
@MikePaquette, makes sense to me. Thanks for the thoughtful response! @FrankHassanabad, feel free to ignore my comment.